admin/webUI/app/components/invite/inviteController.js

```diff
@@ -214,6 +216,21 @@
 		inviteService.getInvitedCandidates(candidatesVm.quizID).then(function(data){
 			candidatesVm.quizCandidates = data.quiz[0]["quiz.candidate"];
 
+			for (var i = 0; i < candidatesVm.quizCandidates.length; i ++) {
+				cand = candidatesVm.quizCandidates[i]
```

Comment:
Try to use semicolon as much as possible, else it will break in minification of js, and use `var cand` for local variable definition.